.github/workflows/docker_build.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           else
             tag=${{ matrix.TF_VERSION }}-${{ matrix.TG_VERSION }}
           fi
-          echo ::set-env name=TAG::${tag}
+          echo "TAG=${tag}" >> $GITHUB_ENV
```

Comment:
Is this the official fix supported by GitHub? Do you also know when set-env will be disabled?

Reply:
@cytopia yes this is the official fix from [GitHub](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#environment-files) and its already disabled as per the workflow was failing because of this